configure.ac

```diff
@@ -20,7 +20,13 @@ AM_INIT_AUTOMAKE([subdir-objects])
 AM_SILENT_RULES([yes])
 AM_MAINTAINER_MODE
 
-AM_CONFIG_HEADER(src/config.h)
+AC_CONFIG_HEADERS([src/config.h])
+
+AC_CHECK_HEADER([winsock2.h], 
```

Comment:
Is this something we have to guard to only be used in win32 environments? I'm not familiar with autotools+windows.

Reply:
It works on my Mac and Linux build systems.  It is just checking for the location of a header, and if found, setting some environment variables.  In the case of non-windows systems, it does not find the header so the vars are empty.